test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenTests.cs

```diff
@@ -14,12 +14,7 @@
 using Microsoft.IdentityModel.Json;
 using Microsoft.IdentityModel.Json.Linq;
 
-#if NET452
-using JsonReaderException = Microsoft.IdentityModel.Json.JsonReaderException;
-#else
-using System.Text.Json;
 using JsonReaderException = System.Text.Json.JsonException;
```

Comment:
this line remains?

Reply:
System.Text,Json is an unnecessary using.